mega-zep-backend/src/test/java/com/gepardec/mega/service/impl/EmployeeServiceImplTest.java

```diff
@@ -119,10 +124,46 @@ void testUpdateEmployeesReleaseDate_EmployeesNotEmpty_EmployeOk() {
             return null;
         }).when(managedExecutor).execute(Mockito.any());
 
-        final List<String> result = beanUnderTest.updateEmployeesReleaseDate(EmployeeTestUtil.createEmployees(1));
+        final List<String> result = beanUnderTest.updateEmployeesReleaseDate(Collections.singletonList(createEmployee(0)));
         Assertions.assertNotNull(result);
         Assertions.assertTrue(result.isEmpty());
     }
 
+    private Employee createEmployee(final int userId) {
```

Comment:
this method has the same code as createEmployeeWithActive, overloaded method could be used